src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -31,7 +31,7 @@
 WINDOWS_ASP_LOCATION_CHINACLOUD_WEBAPP = 'chinaeast'
 
 
-class FunctionappACRScenarioTest(ScenarioTest):
+class FunctionappACRScenarioTest(LiveScenarioTest):
```

Comment:
Had to turn these tests into LiveScenarioTests since they work when run with the `--live` flag but when using the recordings, somehow `azdev test` fails even though the API responses in the yaml files actually show our checks are correct.

Reply:
These tests are not actually impacted by the changes in custom.py so I am not sure why they suddenly started failing.

Reply:
Could you please show me the error message when this test is `ScenarioTest` ?

Reply:
This is the error message:
`azure.cli.testsdk.exceptions.JMESPathCheckAssertionError: Query '[?name=='DOCKER_REGISTRY_SERVER_USERNAME']|[0].value' doesn't yield expected value 'functionappacrtest000004', instead the actual value is 'None'.`

It appears it is trying to use the response from the first `config appsettings list` call instead of the second call.

Reply:
Out of curiosity, could you confirm that running with the dev branch (excluding the code of this PR) also has the same issue? If you can confirm that this issue was not introduced by this PR, then we can skip this issue first and fix it when you have time conditioning to investigate and solve this problem in the future

Reply:
@zhoxing-ms Just made changes. We really just need to remove the AzureWebJobsDashboard app setting for flex apps. After making that change and rerunning the tests, we no longer have issues when they are `ScenarioTest` so I switched them back to `ScenarioTest` from `LiveScenarioTest`.